storage/container_storage_interface/persistent-storage-csi-migration.adoc

```diff
@@ -6,13 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} provides automatic migration for supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers.
-
-:FeatureName: CSI automatic migration
-include::snippets/technology-preview.adoc[leveloffset=+1]
+In-tree storage drivers that are traditionally shipped with {product-title} are being deprecated and replaced by their equivalent Container Storage Interface (CSI) drivers. {product-title} provides automatic migration for certain supported in-tree volume plug-ins to their equivalent CSI drivers.
 
 include::modules/persistent-storage-csi-migration-overview.adoc[leveloffset=+1]
 
+=== Additional resources
+* xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration-automatic-ga[Automatic migration of in-tree volumes to CSI]
+
+* xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration-enable_persistent-storage-csi-migration[Manually enabling CSI automatic migration]
+
```

Comment:
The additional resources pages refers to itself, which is kinda weird.

Reply:
I know - but I find it strange that we need to create "Additional Resources" link which refers to same page. Typically I would expect such links to take me somewhere else and provide more details.

Reply:
I can remove those xrefs if you prefer, but this is the only way to get around the fact that we cannot put xrefs in modules. The summary table refers to those subsections in text, but I cannot make those xrefs unfortunately.